Blog/sammoss.md

```diff
@@ -0,0 +1,8 @@
+#Jan 26
+
+The Public Humanities 
+
+The very personal nature of Risam's article resonated with me, because while not a person of color, I can empathize with some, if not all, of the barriers to coding. I appreciate her framing the discussion in the context of the Torn Apart/Separados projects, because she provides a window into the experience of just jumping in and trying to figure 'it' out. Actually, she resonates hard because she describes some of my own thoughts ("How do I get the code into a thingy"? for one among many. Also, did she really need to use it as an excuse to watch Marvel "yes, including Thor".) She made me laugh and be inspired. Seriously, though, I return to the quandary she presents - the double and triple binds of computational ethic studies. Critically, she made me aware of these issues in the field, so that I can move forward in this space with more awareness of the barriers faced by my peers and help institute change in the moments in which that's within my power. 
+I know Corbett's article is really a sidenote to this, but I just want to add, they also resonated with me because I actually worked ETDs in the Graduate Shool of my alma mater, Univ of North TX, one of the big Tier 1 institutions in TX. The libraries are absolutely outstanding... but they did nothing with ETDs except file them at the time (we were working on more collab). So as the part-time assistant to the severely overworked Grad Reader for the NTX System, I was constantly fielding concerns and questions about Proquest (always looking to make a buck), copyright (which Proquest paperwork tries to make you think you need to pay to be copyrighted). That's all. Hopefully the phenomenal library system has been able to take more of a role so my counterpart is not having bad ETD dreams. 
```

Comment:
Wow, you have first-hand experience of ETDs! I mostly have my own experience of going through the process of filing the dissertation, it's such an interesting niche that wields a lot of power.